src/firmware_new/rules.bzl

```diff
@@ -1800,19 +1536,6 @@ def stm32h7_hal_library_files_genrule(name):
             "Middlewares/Third_Party/FatFs/src/drivers/usbh_diskio_template.h",
             "Middlewares/Third_Party/FatFs/src/drivers/sd_diskio_dma_template.h",
             "Middlewares/Third_Party/FatFs/src/ff.h",
-            "Middlewares/Third_Party/LibJPEG/include/jmorecfg_template.h",
```

Comment:
just curious why these files aren't needed

Reply:
we don't intend on manipulating images on the stm32/run the gui tools w/ the LCD so I just removed the extra stuff that we _for sure_ will not need

also I _think_ it reduces our code size, but I'm not sure if the build system is smart about unused headers

Reply:
ok nvm the unused headers are optimized away, this change is just to reduce clutter then